docker/pypi/wmagent-mariadb/manage

```diff
@@ -149,17 +149,10 @@ init_mariadb(){
     echo "$FUNCNAME: Creating WMAgent user and setting grants"
     # try to create a user different than root (if it does not already exist), and grant privileges
     # we need ${MDB_USER}'@'127.0.0.1 user in paralel to ${MDB_USER}'@'localhost
-
-    if [[ $MDB_USER == $MDB_ROOT ]]; then
-        echo "$FUNCNAME: WARNING: WMAgent user set is the same as the MariaDB Root user. You must configure a different one!"
-        echo "$FUNCNAME: WARNING: NOT creating WMAgent users and NOT granting priveleges to $WMA_DATABASE database"
-        return 1
-    else
-        mariadb -u $MDB_ROOT --password=$MDB_ROOTPASS --socket=$MDB_SOCKET_FILE --execute "CREATE USER '${MDB_USER}'@'localhost' IDENTIFIED BY '$MDB_PASS'"
-        mariadb -u $MDB_ROOT --password=$MDB_ROOTPASS --socket=$MDB_SOCKET_FILE --execute "GRANT ALL ON $WMA_DATABASE.* TO $MDB_USER@localhost"
-        mariadb -u $MDB_ROOT --password=$MDB_ROOTPASS --socket=$MDB_SOCKET_FILE --execute "CREATE USER '${MDB_USER}'@'127.0.0.1' IDENTIFIED BY '$MDB_PASS'"
-        mariadb -u $MDB_ROOT --password=$MDB_ROOTPASS --socket=$MDB_SOCKET_FILE --execute "GRANT ALL ON $WMA_DATABASE.* TO $MDB_USER@127.0.0.1"
-    fi
+    mariadb -u $MDB_ROOT --password=$MDB_ROOTPASS --socket=$MDB_SOCKET_FILE --execute "CREATE USER '${MDB_USER}'@'localhost' IDENTIFIED BY '$MDB_PASS'"
```

Comment:
Please revert the changes  to the `init_mariadb()` function. They are absolutely not related to the current issue and there are no driving reasons for them to happen (neither direct nor indirect) in the fact that we are about to change the user inside the container to be estimated at runtime. On top of that, they are also in contradiction with the decision we have already made in that  regards  months ago. And they are about to break something which was already heavily tested and is working rock solid, just as expected.